lib/algolia/index.rb

```diff
@@ -264,6 +264,56 @@ def get_objects(objectIDs, attributes_to_retrieve = nil, request_options = {})
       client.post(Protocol.objects_uri, { :requests => requests }.to_json, :read, request_options)['results']
     end
 
+    #
```

Comment:
Those docs explain how the method is implemented, and not how should people work with the method. Check the other examples on the file.